CONTRIBUTING.md

```diff
@@ -60,6 +60,23 @@ go test -fuzz=FuzzBooleanEvaluation ./e2e/evaluation_fuzz_test.go
 ```
 substituting the name of the fuzz as appropriate.
 
+### Commit messages
```

Comment:
PR title format

Reply:
I just caught the DCO thing as well, will add